modules/renter/files.go

```diff
@@ -96,6 +96,19 @@ func (f *file) Nickname() string {
 	return f.Name
 }
 
+// Filesize returns the size of the file.
+func (f *file) Filesize() uint64 {
+	lockID := f.renter.mu.RLock()
+	defer f.renter.mu.RUnlock(lockID)
+	// TODO: this will break when we switch to erasure coding.
```

Comment:
it'll break when we start adding multiple files to a single file contract as well.